services/collaboration/pkg/config/http.go

```diff
@@ -0,0 +1,14 @@
+package config
+
+import (
+	"github.com/owncloud/ocis/v2/ocis-pkg/shared"
+)
+
+// HTTP defines the available http configuration.
+type HTTP struct {
+	Addr      string                `yaml:"addr" env:"COLLABORATION_HTTP_ADDR" desc:"The external address of the HTTP service. Either IP address or host (127.0.0.1:9301 or wopi.private.prv). The configured "Scheme" will be used to build public URLs along with this address."`
+	BindAddr  string                `yaml:"bindaddr" env:"COLLABORATION_HTTP_BINDADDR" desc:"The bind address of the HTTP service."`
```

Comment:
Any hint what to enter there? how can one identify/know the target?

Reply:
`The bind address of the HTTP service. Use "<ip-address>:<port>", for example, 127.0.0.1:9301 or 0.0.0.0:9301`

Is that better?

Reply:
Yes, much better.
How does one know/identify this address/port?
Which rules apply?

Reply:
I'd say that it has the same rules as any other service's "Addr".

The problem is that most of the services run internally without external exposure, so using the "Addr" var as a binding address is fine. However, this particular service needs to be available from the outside, so we need a different var.

Maybe it's more clear if we change the variable name: current `Addr` to `ExternalAddr`, and `BindAddr` to `Addr` so it's more aligned with the variables of other services.